WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -65,7 +65,8 @@ private extension AddProductCoordinator {
                                                        productImageActionHandler: productImageActionHandler,
                                                        currency: currency,
                                                        presentationStyle: .navigationStack,
-                                                       isEditProductsRelease3Enabled: true)
+                                                       isEditProductsRelease3Enabled: true,
+                                                       isEditProductsRelease5Enabled: true)
```

Comment:
Please, pay attention to these details, or we risk to enable a feature that is still a work in progress to our users. This should be set to `ServiceLocator.featureFlagService.isFeatureFlagEnabled(.editProductsRelease5)` 👍

Reply:
I am so sorry. I didn't know that. I thought `true ` was OK since the other one was `true `
Updated accordingly